vb-fe/src/components/playlist/Playlist.tsx

```diff
@@ -128,7 +130,7 @@ export const Playlist: FunctionComponent = (): JSX.Element => {
     );
   };
 
-  return !playlistLoading ? (
+  return !playlistLoading ? ( // eslint-disable-line
```

Comment:
What's the `eslint` error here?  Just curious, I don't recognize the situation.  Nested terniaries maybe?

Reply:
Yep nested ternaries aren't allowed. I can make them separate but just wanted to get it done. We can do this in a refactor run later